common-ui/tslint.json

```diff
@@ -60,6 +61,7 @@
       true,
       "ignore-params"
     ],
+    "no-irregular-whitespace": true,
```

Comment:
Note: while you're here, a style inconsistency that has been grating on me is our mixture of absolute and relative module imports. I haven't tried it out, but I found `no-relative-imports` which might alleviate this. Anyways - if that rule is controversial or doesn't work as expected, no need to touch it in this PR.

Reply:
+1 to adding that rule

Reply:
Hmm I don't see the new rule here (but I did see you updated all the import paths - thanks!). Am I missing something?

Reply:
Because it requires an external import, and there's no `package.json` here, I put the rule in `ui/tslint.json`, where the import actually happens.

Reply:
Ah - I see it now. Thanks.